pages/create/index.page.js

```diff
@@ -1,72 +1,113 @@
+import { number, object, string, date, ref } from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useForm } from "react-hook-form";
+
 import Button from "../../components/button/button";
 import Input from "../../components/input/input";
 import Select from "../../components/select/select";
-import { Container, Form, Label, Title } from "./create.styles";
-import { useForm } from "react-hook-form";
-import { number, object, string, date, ref } from "yup";
-import { yupResolver } from "@hookform/resolvers/yup";
+
+import { Container, Form, Title } from "./create.styles";
 
 const Create = () => {
   const validationSchema = object().shape({
-    projectName: string().required(),
+    title: string().required(),
     subtitle: string().required(),
-    category: object().required(),
-    description: string().required(),
-    currency: object().required(),
+    story: string().required(),
+    token: string().required(),
     fundGoal: number().required(),
     startDate: date()
       .required()
-      .min(new Date(), "Created date can not be future"),
+      .max(new Date(), "Created date can not be future"),
     endDate: date()
       .required()
       .min(ref("startDate"), "End date should be after start date"),
```

Comment:
Don't know about yup but seems that endDate accepts a value equals than startDate, could it be?

I suppose that it should be min(ref("startDate") + 1) or something, to force the >= condition